string name
ServoInfo info
---
bool success
string extra_info